language/evm/hardhat-examples/test/FortyTwo.test.js

```diff
@@ -0,0 +1,25 @@
+const { expect } = require("chai");
```

Comment:
Right now, each Move contract and its Solidity equivalent share the same name and are used by the same test script. I don't know if this is a good thing or not...

Reply:
The upside of the shared test script is that we maintain only one test script per contract. Moreover, it shows that a Move contract can replace a Solidity contract, and they are interchangeable. The downside is that some hidden "build cache" affects the workflow, so `npx hardhat clean` is needed before each "compile and test". It's included in the scripts already. Moreover, we may want a scenario in the future that Move contracts and Solidity contracts coexist in the same hardhat project, so that they cross-contract-call each other.